routes/index.mjs

```diff
@@ -1,6 +1,28 @@
 import { default as express } from 'express';
+import passport from 'passport';
 export const router = express.Router();
-import { getAllBooks } from '../controllers/index.mjs';
+import { getAllBooks, getRegister, getLogin, logout } from '../controllers/index.mjs';
 
-/* GET home page. */
-router.get('/', getAllBooks);
+router.get('/', (req, res, next) => {
+  if (req.isAuthenticated()) return next();
```

Comment:
This could be used as a middleware function in other file

Reply:
Yeah I need to structure better the project with middlewares